README.md

```diff
@@ -1,29 +1,37 @@
-# address-index-data [![Build Status](https://travis-ci.com/ONSdigital/address-index-data.svg?token=wrHpQMWmwL6kpsdmycnz&branch=develop)](https://travis-ci.com/ONSdigital/address-index-data)
+# address-index-data 
 
-### Prerequisites
+[![Build Status](https://travis-ci.com/ONSdigital/address-index-data.svg?token=wrHpQMWmwL6kpsdmycnz&branch=develop)](https://travis-ci.com/ONSdigital/address-index-data)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/83c0fb7ca2e64567b0998848ca781a36)](https://www.codacy.com/app/Valtech-ONS/address-index-data?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/address-index-data&amp;utm_campaign=Badge_Grade)
 
-* Java 6 or higher
-* SBT (http://www.scala-sbt.org/)
+### Purpose
 
-### Development Setup (MacOS)
+This repository contains the Scala code for an Apache Spark job to create an Elasticsearch index from the AddressBase premium product.
```

Comment:
Should probably say that it will also work with the Exeter subset that is publicly available. Might get more forks that way?